addons/hr_work_entry_holidays/tests/common.py

```diff
@@ -1,91 +1,11 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from datetime import datetime
-
-from odoo import Command
-from odoo.fields import Datetime
 from odoo.addons.hr_work_entry_contract.tests.common import TestWorkEntryBase
+from odoo.addons.hr_holidays_contract.tests.common import TestHolidayContract
```

Comment:
Are you adding `hr_holidays_contract` as an explicit dependency for `hr_work_entry_holidays`?

Reply:
Hello @jjmaksoud,
We moved common.py code from `hr_work_entry_holidays` to `hr_holidays_contract` to avoid repeating it in both places. When you install `hr_work_entry_holidays`, `hr_holidays_contract` is also installed automatically.